source/_integrations/nest.markdown

```diff
@@ -31,6 +31,8 @@ There is currently support for the following device types within Home Assistant:
 - [Climate](#climate)
 - [Sensor](#sensor)
 
+Currently the New Nest Cams and the New Nest Doorbell (battery) arent supported for third party applications.
```

Comment:
This should be added under the `- Display, Camera, and Doorbell Devices` bullet point in the  `Overview: Supported Devices` section